guides/v2.0/contributor-guide/templates/basic_template.md

```diff
@@ -9,41 +9,41 @@ github_link: contributor-guide/templates/basic_template.md
 redirect_from: contributor-guide/basic_template.html
 ---
 
-### Overview
+## Overview
 Introductory text that gives an overview of the topic you will be writing about.
 
 The purpose of this page is to provide you with a pre-formatted template and useful {% glossarytooltip a5ef9041-976f-4eb3-826e-bf836027d8c3 %}markdown{% endglossarytooltip %} references to help you get started writing docs.
 
 You can start off by editing the local version of this file using markdown language (and {% glossarytooltip a2aff425-07dd-4bd6-9671-29b7edefa871 %}HTML{% endglossarytooltip %} where needed). Then, create a Pull Request to have your contribution reviewed by the DevDocs team.
 
-### Basic Markdown Syntax
+## Basic Markdown Syntax {#basic}
 Below are some basic examples of what you can do with markdown.
 
-#### Text Effects
+### Text Effects {#text}
 
 | Example           | Output       |
 | ----------------- | ------------ |
 |`*emphasis*`       |*emphasis*    |
 |`**bold**`         | **bold**     |
 |`` `inline code` ``| `inline code`|
 
-By indenting your content by at least 4 spaces, you can create a code block.
+By indenting your content by at least 4 spaces, you can create a code block. For extensive examples of adding code samples, see [Code blocks](#code-blocks).
 
     //This is a code block!
     print "Hello World!";
 
-For more examples of basic markdown please follow this [link](https://daringfireball.net/projects/markdown/syntax){:target="_self"}.
+For more examples of basic markdown please follow this [link](https://daringfireball.net/projects/markdown/syntax){:target="_blank"}.
 
-#### Lists
+### Lists {#lists}
 Lists are useful for organizing and displaying related items. Below are examples of a bulleted list and an ordered list.
 
 **Bulleted List:**
 
```

Comment:
You don't have to add code formatting to a markdown list, right? Confused why this is here.

Reply:
This just shows a sample in rouge markdown for creating a bullet list. In the generated content, it doesn't show "markdown", just shows the code sample with the generated content after. Just cleaned up the way we use code samples throughout.